Add April news template

cticenhour opened the pull request:
Also fixes up a couple things on March news 

Refs #11447

cticenhour committed:
Add April news template, and fixup March news (#11447)

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/996935/) on 52a46ac wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/20652/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/996943/) on 52a46ac wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>fa23b0</th>
      <th colspan=4>#20652 52a46a</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>82.55%</td>
      <td>82.55%</td>
      <td>+0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>73723</td>
      <td>73724</td>
      <td>+1</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>15580</td>
      <td>15579</td>
      <td>-1</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/20652/coverage_diff/framework.html)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/20652/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/contact)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/richards)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/20652/coverage/xfem)

</details>



This comment will be updated on new commits.

aeslaughter committed:
Add news about MOOSE IGA (refs #11446)

aeslaughter committed:
Update large_media (refs #11446)

aeslaughter committed:
Apply suggestions from code review

Co-authored-by: Casey Icenhour <cticenho@ncsu.edu>

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit acc498685d1453fda11c2000886fdacdae6ae333
+Subproject commit 4186a099a5a7cd2f865a404dafbfcac74a3bd832
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

roystgnr committed:
Discuss Exodus IGA support more explicitly

The `clear_spline_nodes` setting should go in via #20646 any time now;
everything else is working already and was enabled for most users by the
HDF5 updates.
